### PR TITLE
Fix CentOS 7 integration tests

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   integration:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (tyll) [inline .github/workflows/integration.yml:12]: Please provide an explanation to the commit message why this is the correct change.

Comment (liangwen12year) [reply .github/workflows/integration.yml:12]: Updated.

**Review by tyll — APPROVED** [review]
:+1: LGTM

**Comment by liangwen12year** [comment]
Notice that, this PR will make all the c7, c8s, c9s integration tests  run on the ubuntu-20.04 runner image, which is not the ideal case. Because we still want the c8s, c9s integration tests  run on the ubuntu-latest runner image.  Later on, we may add patch to improve it.

**Pull request merged**